contract-map.json

```diff
@@ -3197,6 +3197,14 @@
     "symbol": "CR",
     "decimals": 8
   },
+  "0xF94b5C5651c888d928439aB6514B93944eEE6F48": {
```

Comment:
Before I will be able to merge Kanpeki please remove this one as well as yld file and open this in new PR. I can not make a merge for overlapped symbols in this repo